demo/src/pages/when.astro

```diff
@@ -1,34 +1,33 @@
 ---
-import Document from '../layouts/Document.astro'
-import { When } from '@astropub/flow'
+import Document from "../layouts/Document.astro";
+import { When } from "@astropub/flow";
```

Comment:
General Nit / Tip: For PR review, it can be helpful to follow the formatting conventions of the existing project. For example, when I’m PR’ing a project that uses 2-spaces or 4-spaces, then I use 2-spaces or 4-spaces, and if the project uses tabs then I use tabs. In this case, the changes to strings distract from the improvements you’re looking to highlight.

I am super interested to see the parts where you really wanted to change how things work.

Reply:
The changes were due to prettier. I think it's better for me to take it out of the project